apps/tailwind-components/app/components/button/PageHeader.vue

```diff
@@ -0,0 +1,26 @@
+<script lang="ts" setup>
```

Comment:
not sure if we need a page header button , it there a design for this ? current design includes a backbtn ( icon only pointing left), if thats is the button that is need i would suggest only a direct impl of that specific btn or create a specific form header that allows for a back btn